### PR TITLE
Update testing.md

### PR DESCRIPTION
**Pull request by icarusdust**

### CODE REVIEW

File: content/yaml-testing/testing.md [hunk 1]
```diff
@@ -49,8 +49,8 @@ scripts:
 ### Running iOS/Android application tests on a mobile simulator/emulator
 
 ```bash
```
Comment (nevercode-kevin) [inline content/yaml-testing/testing.md:51]:
```suggestion
```

File: content/yaml-testing/testing.md [hunk 2]
```diff
@@ -49,8 +49,8 @@ scripts:
...
-flutter emulators --launch apple_ios_simulator &                    # for android use: flutter emulators --launch emulator
-adb wait-for-device 
+flutter emulators --launch apple_ios_simulator &            # for android use: flutter emulators --launch emulator
+adb wait-for-device                                         # adb wait-for-device is for android use only
 flutter drive --driver=test_driver/integration_driver.dart --target=integration_test/app_test.dart -d iPhone  # for android use: -d emulator-5554
 ```
```
Comment (nevercode-kevin) [inline content/yaml-testing/testing.md:55]:
```suggestion
```

File: content/yaml-testing/testing.md [hunk 3]
```diff
@@ -49,8 +49,8 @@ scripts:
 ### Running iOS/Android application tests on a mobile simulator/emulator
```
Comment (nevercode-kevin) [inline content/yaml-testing/testing.md:50]:
```suggestion
For the Android emulator you can launch and run your tests as follows:

```yaml
flutter emulators --launch emulator &           # The ampersand is used to run the emulator in the background without blocking the next command
adb wait-for-device                                         # adb wait-for-device is used to wait for the emulator to finish loading
flutter drive --driver=test_driver/integration_driver.dart --target=integration_test/app_test.dart -d emulator-5554
```

You can launch the iOS simulator and run tests on the simulator as follows:

```yaml
flutter emulators --launch apple_ios_simulator
flutter drive --driver=test_driver/integration_driver.dart --target=integration_test/app_test.dart -d iPhone 
```
```

**Review by nevercode-kevin — CHANGES_REQUESTED** [review]
Try this suggestion. Might need formatting better, best to test locally.

**Comment by icarusdust** [comment]
Added the changes!

**Review by nevercode-kevin — APPROVED** [review]

**Pull request merged**